guide/predict_cli.ipynb

```diff
@@ -114,9 +114,12 @@
     "In v0.12.6, we introduced the ability to pipe the output to a virtual camera.\n",
     "This virtual camera can then be used as the source camera in Zoom and other\n",
     "conferencing softwares. You need a virtual camera on your system, e.g.\n",
-    "from [OBS Studio](https://obsproject.com) and need to \n",
-    "install `pip3 install pyvirtualcam`. Then you can use the \n",
-    "`--video-output=virtualcam` argument."
+    "from [OBS Studio](https://obsproject.com) (Mac and Windows) or\n",
+    "[v4l2loopback](https://github.com/umlaeute/v4l2loopback) (Linux)\n",
+    "and need to install `pip3 install pyvirtualcam`. Then you can use the \n",
+    "`--video-output=virtualcam` argument.\n",
+    "<!--- Comment from Duncan: Install on Linux distributions with 'sudo apt install v4l2loopback-dkms'\n",
```

Comment:
This is like a footnote in a paper and a footnote is almost always bad style: the information is either important enough to include in the main text or it is not necessary.

My preference would to remove this comment and substitute the above link with `https://github.com/umlaeute/v4l2loopback#distributions` which shows this install command.